iron-checked-element-behavior.html

```diff
@@ -64,6 +65,10 @@
       '_requiredChanged(required)'
     ],
 
+    created: function() {
+      this._hasIronCheckedElementBehavior = true;
```

Comment:
I would maybe add a comment about why this is needed. It doesn't seem to be used anywhere in this class, so I would totally be tempted to nuke it in the future.

Also, I think you might want to define this in the properties block, or else `polylint` will complain that you're accessing a non-defined property (in iron-form, where you're actually using it)